custom-elements/reactions/CSSStyleDeclaration.html

```diff
@@ -151,8 +158,24 @@
 }, 'cssFloat on CSSStyleDeclaration must not enqueue an attributeChanged reaction when it adds the style attribute but the style attribute is not observed');
 
 test_mutating_style_property_value(function (instance, propertyName, idlName, value) {
-    instance.style[idlName] = value;
-}, 'camel case property on CSSStyleDeclaration');
+    assert_equals(idlName, 'borderWidth');
+    instance.style.borderWidth = value;
+}, 'A camel case attribute (borderWidth) on CSSStyleDeclaration', {propertyName: 'border-width', idlName: 'borderWidth', value1: '2px', value2: '4px'});
+
+test_mutating_style_property_value(function (instance, propertyName, idlName, value) {
+    assert_equals(propertyName, 'border-width');
+    instance.style['border-width'] = value;
+}, 'A dashed property (border-width) on CSSStyleDeclaration', {propertyName: 'border-width', idlName: 'borderWidth', value1: '1px', value2: '5px'});
+
+test_mutating_style_property_value(function (instance, propertyName, idlName, value) {
+    assert_equals(idlName, 'webkitWritingMode');
```

Comment:
-webkit-writing-mode is not supported across browsers and isn't in https://compat.spec.whatwg.org/

How about -webkit-border-radius instead (is supported in WebKit/Chromium/Gecko/Edge 13)?